proxy/src/avatar.rs

```diff
@@ -19,7 +19,7 @@ const EMOJIS: &[&str] = &[
 
 /// An emoji.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
-pub struct Emoji(&'static str);
+pub struct Emoji(pub &'static str);
```

Comment:
This kind of defeats the purpose of a newtype, I would rather we implement `Display` on it. Then you can call `to_string()`.

Reply:
Addressed in 0fc506a.